varats/varats/plots/blame_diff_correlation_matrix.py

```diff
@@ -164,6 +212,11 @@ def plot(self, view_mode: bool) -> None:
             raise PlotDataEmpty
         df.sort_values(by=['time_id'], inplace=True)
 
+        for x_var in variables:
+            for y_var in variables:
+                if x_var != y_var:
+                    log_interesting_revisions(x_var, y_var, df.copy())
+
```

Comment:
```suggestion
        if LOG.getLogger().isEnabledFor(LOG.INFO):
            for x_var in variables:
                for y_var in variables:
                    if x_var != y_var:
                        log_interesting_revisions(x_var, y_var, df.copy())

```
This should only run during logging